Moderation-Policy.md

```diff
@@ -27,10 +27,9 @@ supported by the Admin team of the Slack organization.
 
 ## Terms
 
-* *Collaborator* refers to any individual with configured write (commit)
-  permissions to any Node.js GitHub organization repository *other than the
-  Moderation Repository*. See [GitHub's access permissions documentation][] for
-  more information.
+* *Collaborator* refers to any individual with configured triage permissions to
```

Comment:
Nit: Maybe make this slightly clearer by saying "triage or higher" or something like that?

Reply:
I guess it should say "Triage role or higher", since GitHub does not define a "Triage permission" 🤔